arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -198,6 +198,13 @@
           "type": "string",
           "description": "The run command id."
         },
+        "script": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The script to be executed."
```

Comment:
I'm not familiar with the API, so maybe the question is stupid, on the other hand new customers may feel the same way:

How does this new property relate to the others? Since it wasn't there before, I assume `commandId` would identify the command to run - so what happens if `script` is specified? Is it something that runs before/after the command? Is it something that runs instead of the command identified by `commandId`? Are you supposed to only provide *either* `commandId` *or* `script`?

I'm open to being convinced why new SDK users would know this. Otherwise, please add sufficient information to the descriptions. 😉